ebean-migration/src/test/java/dbmig/V1_2_1__test.java

```diff
@@ -2,31 +2,22 @@
 
 import java.sql.Connection;
 
-import io.ebean.migration.ConfigurationAware;
 import io.ebean.migration.JdbcMigration;
 import io.ebean.migration.MigrationConfig;
 
 /**
  * Sample migration.
  *
  * @author Roland Praml, FOCONIS AG
- *
  */
-public class V1_2_1__test implements JdbcMigration, ConfigurationAware{
-
-  private MigrationConfig config;
+public class V1_2_1__test implements JdbcMigration {
 
   public static class MyDto {
     String id;
   }
-  
-  @Override
-  public void setMigrationConfig(MigrationConfig config) {
-    this.config = config;
-  }
 
   @Override
-  public void migrate(Connection connection) {
+  public void migrate(Connection connection, MigrationConfig config) {
```

Comment:
In a more ideal API some interface would be here. It would be read-only (only getters/accessors).

e.g. `JdbcMigrationContext` with methods like:
- Connection connection() 
- Transaction transaction() // return the connection wrapped as a ebean transaction
- Platform platform() // ??

What is in MigrationConfig that JdbcMigration implementations need to access?